test/index.ts

```diff
@@ -410,13 +420,13 @@ describe('BigQuery', () => {
       day: 1,
     };
 
-    it('should expose static and instance constructors', () => {
+    it.skip('should expose static and instance constructors', () => {
```

Comment:
@stephenplusplus @callmehiphop so this is broke.  Instead of `bigquery.date` referring to a type, a static method, and an instance method - I'm now just exporting `BigQueryDate` as a type (same for the other similar objects), and keeping the instance and static methods. 

Thoughts on the approach?

Reply:
gentle ping @stephenplusplus @callmehiphop on your take here :)

Reply:
Also, if this is a change in API, please document what was changed and how users should update their code